images/egress/dns-proxy/egress-dns-proxy.sh

```diff
@@ -80,12 +80,35 @@ function generate_dns_resolvers() {
     echo ""
 }
 
+function get_num_servers() {
+    local domain=$1
+
+    if [[ "${EGRESS_DNS_PROXY_MODE:-}" == "unit-test" ]]; then
+        if [[ "${DNS_SERVERS[${domain}]:-}" != "" ]]; then
+            echo "${DNS_SERVERS[${domain}]}"
+        else
+            echo "1"
```

Comment:
This will work, but it would be simpler to use `echo "${DNS_SERVERS[${domain}]:-1}"`.

Reply:
done